modules/google_analytics_analyzer/config/install/views.view.analize_google_analytics.yml

```diff
@@ -0,0 +1,822 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - google_analytics_reports
+    - google_analytics_analyzer
+id: analize_google_analytics
+label: 'Analize Google Analytics'
```

Comment:
Should be anal**y**ze

Reply:
Fixed in all places, including the actual filename